core/src/main/java/org/eclipse/hono/config/ProtocolAdapterProperties.java

```diff
@@ -36,7 +36,7 @@
     private long eventLoopBlockedCheckTimeout = 5000L;
     private int maxConnections = 0;
     private Duration tenantIdleTimeout = DEFAULT_TENANT_IDLE_TIMEOUT;
-    private Map<String, MapperEndpoint> mapperEndpoints = new HashMap<>();
+    private HashMap<String, MapperEndpoint> mapperEndpoints = new HashMap<>();
```

Comment:
We usually use an abstract type for field declarations, don't we? Why can't we keep the `Map` here?

Reply:
We're basically hitting this with using the Map: https://github.com/quarkusio/quarkus/issues/8017. And that's not considered as the issue at the moment. We can create our own converter perhaps and I'll ask if there are a better way to do all this, but to me this is a quick/easy workaround that we can use for now.